resources/frameworks.yaml

```diff
@@ -113,7 +113,9 @@ TPOT:
 #    population_size: 25
 #    verbosity: 2
 
-
+lightautoml:
+  project: https://github.com/sberbank-ai-lab/LightAutoML
+  version: '0.2.5'
```

Comment:
new requirements:
- please insert your framework in alphabetical order in the frameworks definitions files.
- `frameworks.yaml` should have version set to `stable` in this file (pointing to last release).
- in `frameworks.yaml`, a short description and references are welcome (see others in that file).
- please also add an entry to new `frameworks_latest.yaml`.